src/commands/jetpack-module.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+namespace Team51\Command;
+
+use Team51\Helper\API_Helper;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Helper\Table;
+use Symfony\Component\Console\Helper\ProgressBar;
+
+
+class Jetpack_Module_Search extends Command {
+	protected static $defaultName = 'module-search';
+
+	/**
+	 * @var Api_Helper|null API Helper instance.
+	 */
+	protected $api_helper = null;
+
+	public function __construct() {
+		parent::__construct();
+
+		$this->api_helper = new API_Helper();
+	}
+
+	protected function configure() {
+		$this
+		->setDescription( 'Searches Team 51 sites for a specified Jetpack module based on its status.' )
+		->setHelp( 'Use this command to show a list of sites where a particular Jetpack module is on or off. This command requires a Jetpack site connected to the a8cteam51 account.' )
+		->addArgument( 'module-slug', InputArgument::REQUIRED, 'The slug of the Jetpack module to search for.' )
+		->addArgument( 'module-status', InputArgument::REQUIRED, 'The status of the Jetpack module to search for.' );
+	}
+
+	protected function execute( InputInterface $input, OutputInterface $output ) {
+		$module_slug   = $input->getArgument( 'module-slug' );
+		$module_status = $input->getArgument( 'module-status' );
+
+		$api_helper = new API_Helper;
+
+		$output->writeln( '<info>Fetching list of sites...<info>' );
+
+		$sites = $api_helper->call_wpcom_api( 'rest/v1.1/jetpack-blogs/', array() );
+
+		if ( empty( $sites ) ) {
+			$output->writeln( '<error>Failed to fetch sites.<error>' );
+			exit;
+		}
+
+		$site_list = array();
+		foreach ( $sites->blogs->blogs as $site ) {
+			$site_list[] = array( $site->userblog_id, $site->domain );
+		}
+		$site_count = count( $site_list );
+
+		$output->writeln( "<info>{$site_count} sites found.<info>" );
+		$output->writeln( "<info>Checking each site for the Jetpack module: {$module_slug}<info>" );
+		$output->writeln( '<comment>"Patience you must have, my young padawan."</comment>' );
+
+		$progress_bar = new ProgressBar( $output, $site_count );
+		$progress_bar->start();
+
+		$module_match_status = array();
+		$sites_not_checked   = array();
+		foreach ( $site_list as $site ) {
+			$progress_bar->advance();
+			$module_list = $this->get_list_of_modules( $site[0] );
```

Comment:
I wonder if the whole process would speed up if we make this call async. 

We do something like that for the `remove-user` command. However, I'm in the process of looking for a better async option because the one we use (`Amphp::parallelMap`) sometimes doesn't work on all computers.

Reply:
That would be helpful since it does take a very long time. It's also the same issue with #75 since it uses the same process. 
Does this then just run that particular function in multiple CPU threads? I'm curious as to how it handles the multiple calls, and could that create issues with API rate limits?

Reply:
> Does this then just run that particular function in multiple CPU threads? 

I think so, but not 100% sure. Sounds like at some point they require to create temporary files on the fly, and that triggers an error on some computers.

This is their main repository: https://github.com/amphp

We use Parallel, but they also have a library dedicated to Async HTTP requests which I plan to give a try.

Reply:
Ok. I think I'll merge as it is for now, along with the command renaming, and we can tackle this later. 
Noting here, I did try several options to speed this up, similar to what you discovered [here](https://github.com/Automattic/team51-cli/pull/77#discussion_r875050753), however it doesn't look like I can filter down through the fields with the `jetpack-blogs` endpoint. 
I'm thinking the delay is happening because of the sequential API calls, one after the other for each site. I was hoping I would have been able to download all of the sites' data in one call, then do the search/comparison on the downloaded data - but as far as I can tell it's not possible without looping through each site ID.

Reply:
> Ok. I think I'll merge as it is for now

Yes, I agree that's a good call.

> I'm thinking the delay is happening because of the sequential API calls, one after the other for each site.

I think the same. This week I'll be working on an implementation for async calls (different than `parallelMap`), I'll ping you when we have a function that you can re-use here @nickpagz